### ISSUE TITLE
Define Trestle OSCAL extensions in a language agnostic way

### ISSUE THREAD

**Issue by jpower432** [oscal-compass/compliance-trestle#1779]

**Comment by butler54** [comment]
I think you are going to need to define these in metaschema?

**Comment by butler54** [comment]
@vikas-agarwal76 discussion: We need to be clearer about documenting these extensions and why.

**Comment by jpower432** [comment]
Yes @butler54. I was thinking we might look at defining `metaschema` constraints to define the extensions https://github.com/usnistgov/metaschema/blob/develop/schema/xml/metaschema-meta-constraints.xsd.

**Comment by jpower432** [comment]
> We need to be clearer about documenting these extensions and why.

@butler54 @vikas-agarwal76 Can you help me understand what exactly needs to be clarified? Does the use case for extension documentation need more expansion?
